src/templates/events/event-template.js

```diff
@@ -116,8 +117,26 @@ export default ({ data, pageContext }) => {
     seo,
   } = frontmatter;
 
+  const todaysDate = new Date();
 
+  const dateString = `${todaysDate.getFullYear()}-${
```

Comment:
you can actually compare two Date objects directly with `>`, `<`, `<=` or `>=`, so there's no need to construct date strings to compare.